crypto/ec/curve448/arch_32/f_impl.c

```diff
@@ -0,0 +1,111 @@
+/*
+ * Copyright 2017-2018 The OpenSSL Project Authors. All Rights Reserved.
+ * Copyright 2014 Cryptography Research, Inc.
+ *
+ * Licensed under the OpenSSL license (the "License").  You may not use
+ * this file except in compliance with the License.  You can obtain a copy
+ * in the file LICENSE in the source distribution or at
+ * https://www.openssl.org/source/license.html
+ *
+ * Originally written by Mike Hamburg
+ */
+
+#include "field.h"
+
+/* Use an unrolled loop for performance */
+#define FOR_LIMB(_i, _start, _end, _x) \
```

Comment:
Thanks, looks good, I would however like to use OPENSSL_SMALL_FOOTPRINT
to Switch to the non-inlined variant.

Reply:
Ok. Done. I moved this define to curve448utils.h so that it could be picked up by f_generic.c too.